packages/example-react/components/Header.js

```diff
@@ -23,4 +23,4 @@ const rule = () => ({
   }
 })
 
-export default createComponent(rule, Header, ['title'])
```

Comment:
You will need to readd the title prop or use the new createComponentFactory or else the Header won't show its title :)